ui/lib/pki/addon/components/page/pki-issuer-details.hbs

```diff
@@ -59,7 +59,11 @@
             </InfoTableRow>
           {{else if (eq attr.name "keyId")}}
             <InfoTableRow @label={{or attr.options.label (humanize (dasherize attr.name))}} @value={{get @issuer attr.name}}>
-              <LinkTo @route="keys.key" @model={{get @issuer attr.name}}>{{get @issuer attr.name}}</LinkTo>
+              {{#if (gt @issuer.keyId 0)}}
```

Comment:
If a key doesn't exist is the value 0? Curious why `{{#if @issuer.keyId}}` doesn't work

Reply:
This was pre-holiday brain 👀 I don't remember why I did it this way, or what I did to make an issuer without a keyId but I'll investigate and report back. I do know that the lack of keyId on the response broke the page because LinkTo expected it